istio.VERSION

```diff
@@ -7,3 +7,4 @@ export MIXER_TAG="1bc30a23190aa58635d02ff7fd31bf74de0d011e"
 export ISTIOCTL_URL="https://storage.googleapis.com/istio-artifacts/pilot/330dd286541d1b84c5ac1f4fc504556796c070af/artifacts/istioctl"
 export PILOT_HUB="gcr.io/istio-testing"
 export PILOT_TAG="330dd286541d1b84c5ac1f4fc504556796c070af"
+export ISTIO_NAMESPACE=istio-system
```

Comment:
@kyessenov this PR isn't changing pilot so let's fix the auth in a simple follow up which should have been done whenever that authPolicy change happened